sdk/search/Azure.Search.Documents/src/Indexes/Models/SearchIndex.cs

```diff
@@ -37,6 +37,7 @@ public SearchIndex(string name)
             Suggesters = new ChangeTrackingList<SearchSuggester>();
             TokenFilters = new ChangeTrackingList<TokenFilter>();
             Tokenizers = new ChangeTrackingList<LexicalTokenizer>();
+            Normalizers = new ChangeTrackingList<LexicalNormalizer>();
```

Comment:
Can we change this .ctor to call the other so we only need to keep one of these up to date over time?